src/switcher-ui.tsx

```
@@ -1918,11 +1946,21 @@ function SwitcherApp() {
                           />
                         </span>
                         {customTitles[session.sessionId] && (
-                          <span style={{ color: '#7ec87e', fontSize: '13px', fontWeight: '500' }}>
+                          <span
```

ESLint

[error] Insert `······`

(prettier/prettier)